build.sbt

```diff
@@ -53,7 +53,7 @@ lazy val coverageSettings: Seq[Setting[_]] = {
 
   Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimumStmtTotal := 95,
+    ScoverageKeys.coverageMinimumStmtTotal := 85,
```

Comment:
After removal of all the unused elements above I was able to get slightly above 90% coverage, so can this be bumped to 90? We will still look to play the future IT test story to get it back up to 95.